plugins/modules/lvg.py

```diff
@@ -418,89 +369,40 @@ def make_vg(module, vg_name, vg_state):
     global result
 
     pvs = module.params['pvs']
-
-    cmd = "mkvg "
+    opt = build_vg_opts(module)
 
     vg_type_opt = {
         "none": '',
         "big": '-B ',
         "scalable": '-S ',
     }
-
     vg_type = module.params["vg_type"]
     if vg_type:
-        cmd += vg_type_opt[vg_type]
-
-    enhanced_con_vg = module.params["enhanced_concurrent_vg"]
-    if enhanced_con_vg:
-        cmd += '-C '
-
-    critical_pvs = module.params["critical_pvs"]
-    if critical_pvs:
-        cmd += '-e y '
-
-    force = module.params["force"]
-    if force:
-        cmd += '-f '
-
-    mpool_opt = {
-        'none': '',
-        'normal': '-M y ',
-        'strict': '-M s ',
-    }
-    mpool_strict = module.params["mirror_pool_strict"]
-    if mpool_strict:
-        cmd += mpool_opt[mpool_strict]
-
-    if module.params["multi_node_vary"] is False:
-        cmd += '-N n '
-
-    if module.params["auto_on"] is False:
-        cmd += '-n '
-
-    if module.params["retry"]:
-        cmd += '-O y '
+        opt += vg_type_opt[vg_type]
 
+    # specify which mirror pool the PVs will be
+    # assigned into on creation of the VG
+    # NOTE: mirror pools are only for scalable VG
+    # this option is ignored in small and big VG
     mirror_pool = module.params["mirror_pool"]
     if mirror_pool:
-        cmd += "-p %s " % mirror_pool
-
-    num_partitions = module.params["num_partitions"]
-    if num_partitions:
-        cmd += "-P %s " % num_partitions
-
-    critical_vg = module.params["critical_vg"]
-    if critical_vg:
-        cmd += "-r y "
+        opt += "-p %s " % mirror_pool
 
```

Comment:
Same comment as before. In the build_vg_opts, the "-r n" will be added for non critical_vg. Is that OK?